handlers/summary.go

```diff
@@ -0,0 +1,106 @@
+package handlers
+
+import (
+	"encoding/json"
+	"github.com/gin-gonic/gin"
+	"github.com/openearthplatforminitiative/agriculture-api/models"
+	"io"
+	"log"
+	"net/http"
+	"net/url"
+)
+
+func Summary(c *gin.Context) (summary models.Summary) {
+	params := c.Request.URL.Query()
+	soilData := getData(params, "soil/type")
+	weatherData := getData(params, "weather/locationforecast")
+	floodData := getData(params, "flood/summary")
+	deforestationData := getData(params, "deforestation/basin")
+
+	return createSummary(soilData, weatherData, floodData, deforestationData)
+}
+
+func getData(params url.Values, endpoint string) (body []byte) {
+	resp, err := http.Get("https://api.openepi.io/" + endpoint + "?lat=" + params.Get("lat") + "&lon=" + params.Get("lon"))
```

Comment:
It is a bit fragile to have the query-params for each service hardcoded like this. Even if all the apis use the same params (lat/lon) it might be different in the future.

As of now, the weather api also supports the altitude param. Perhaps this should be refactored.